charts/consul/templates/connect-inject-clusterrole.yaml

```diff
@@ -31,6 +31,7 @@ rules:
   - peeringacceptors
   - peeringdialers
   {{- end }}
+  - jwtproviders
```

Comment:
I think I got the plurality right here, but there were so many places to update I'm not positive.